Link and owncloud tld fixes (#3850)
owncloud/docs: modules/admin_manual/pages/installation/configuration_notes_and_tips.adoc

passing the needed authentication headers to PHP and so the login to
ownCloud via WebDAV, CalDAV and CardDAV clients is failing. Information
on how to correctly configure your environment can be found
{central-url}/t/no-basic-authentication-headers-were-found-message/819[in

> the forums] but we generally recommend against the use of these modules

Reviewer: ```suggestion
the forums] but we generally recommend not to use these modules
```
